lib/bech32/README.md

```diff
@@ -0,0 +1,27 @@
+# Bech32 library
+
+The library implements Bech32 which is a segwit address format specified by [BIP-0173](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki).
+Basically, it is a checksummed base32 format and a standard for native segregated witness output addresses using it.
+
+# How to use it
+
+See the hello world example below:
+
+``` haskell
+λ import Codec.Binary.Bech32
+λ import Prelude
+λ let (Right hrp) = humanReadablePartFromText "ca"
+λ hrp
+HumanReadablePart "ca"
+λ let addr = "835823f15b7085fe1efb6af305e65da62bcf5226779a91dd3130b155cb697782b1" :: Text
+λ import Data.Text.Encoding
+λ let data = dataPartFromBytes (encodeUtf8 addr)
```

Comment:
Unfortunately, `data` is a reserved keyword, so we can't use it for the name of an identifier.
```suggestion
λ let dataPart = dataPartFromBytes (encodeUtf8 addr)
```

Reply:
indeed, corrected